app/views/rfp_forms/rfp_form.html.erb

```diff
@@ -0,0 +1,146 @@
+      <div class="col-sm-10"> 
+        <input type="tel" class="form-control" name="contact_phone" placeholder="Enter phone" value="<%= @rfp_form.contact_phone %>">
+      </div>
+    </div>
+    <div class="form-group">
+      <label class="control-label col-sm-2" for="contact_email">Email:</label>
+      <div class="col-sm-10"> 
+        <input type="email" class="form-control" name="contact_email" placeholder="Enter email" value="<%= @rfp_form.contact_email %>">
+      </div>
+    </div>
+<hr>    
+<h3 class="form-title well">2. Organization</h3>
+
+    <div class="form-group">
+      <label class="control-label col-sm-2" for="employees">Employees:</label>
+      <div class="col-sm-10"> 
+        <input type="number" class="form-control" name="employees" placeholder="Enter total number" min="0" value="<%= @rfp_form.employees %>" value="<%= @rfp_form.employees %>">
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label class="control-label col-sm-2" for="volunteers">Volunteers:</label>
+      <div class="col-sm-10"> 
+        <input type="number" class="form-control" name="volunteers" placeholder="Enter total number" min="0" value="<%= @frp_form.volunteers %>">
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label class="control-label col-sm-2" for="board">Active Board Members:</label>
+      <div class="col-sm-10"> 
+        <input type="number" class="form-control" name="board" placeholder="Enter total number" min="0" value="<%= @rfp_form.board %>">
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label class="control-label col-sm-2" for="budget">Total annual budget for organization:</label>
+      <div class="col-sm-10"> 
+        <input type="number" class="form-control" name="budget" placeholder="Enter annual budget" min="0" value="<%= @rfp_form.budget %>">
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label class="control-label col-sm-2" for="affiliated">If affiliated with a national or parent organization, what role does the organization play in your strategic direction, operations, and overall governance?</label>
+      <div class="col-sm-10"> 
+        <textarea type="text" class="form-control" rows="7" name="affiliated" placeholder="Role of Affiliations: please limit to 50 words."><%= @rfp_form.affiliated %></textarea>
+      </div>
+    </div>
+<hr>
+
+<h3 class="form-title well">3. Brief History</h3>
+
+    <div class="form-group">
+      <label class="control-label col-sm-2" for="mission">Provide a <em>brief</em> summary of your organization’s history.</label>
+      <div class="col-sm-10"> 
+        <textarea type="text" class="form-control" rows="7" name="history" placeholder="History: please limit to 50 words."><%= @history.mission %></textarea>
+      </div>
+    </div>
+<hr>
+
+<h3 class "form-title well">4. Current Programs and Services</h3>
+
+    <div class="form-group">
+
+
```

Comment:
Oh my gosh Trevor, no. That code was what I pushed up last week for you to see and I accidentally called for a pull request on but have to fix. I am working on the form-for and will have it ready soon. I overbooked this week and am paying for it in terms of lost time.

Sent from my iPhone

> On Mar 5, 2016, at 1:12 PM, Trevor Jones <notifications@github.com> wrote:
> 
> In app/views/rfp_forms/rfp_form.html.erb:
> 
> > +
> > +<h3 class="form-title well">3. Brief History</h3>
> > +
> > +    <div class="form-group">
> > +      <label class="control-label col-sm-2" for="mission">Provide a <em>brief</em> summary of your organization’s history.</label>
> > +      <div class="col-sm-10"> 
> > +        <textarea type="text" class="form-control" rows="7" name="history" placeholder="History: please limit to 50 words."><%= @history.mission %></textarea>
> > +      </div>
> > +    </div>
> > +<hr>
> > +
> > +<h3 class "form-title well">4. Current Programs and Services</h3>
> > +
> > +    <div class="form-group">
> > +
> > +
> @dmvana So it looks like you chose not to use form_for. I would recommend you still learn it - both you and @MVanderwood and @ninaleung. I imagine most employers expect an understanding here, so it would be good to learn and practice. It cleans up the code and allows Rails to work better by binding the object to the form. http://guides.rubyonrails.org/form_helpers.html
> 
> —
> Reply to this email directly or view it on GitHub.
>